client/header/activity-panel/style.scss

```diff
@@ -8,7 +8,8 @@
 	height: $header-height;
 
 	@include breakpoint( '<782px' ) {
-		position: relative;
+		position: absolute;
+		top: 60px;
```

Comment:
Since the header has the potential to change in height, I'd like to avoid setting a fixed height.  I think `top: 100%` should work okay here.